### ISSUE TITLE
Apply rustfmt?

### ISSUE THREAD

**Issue by sunchao** [sunchao/parquet-rs#187]
This has been discussed long time back (see #44). As we have more people contributing to the project now, I wonder if we should revisit this, provide a rustfmt template and check it in CI. Personally, I had some good experiences with rustfmt in other projects, as well as similar stuff in Go. @sadikovi : want to hear you opinion on this.

**Comment by sadikovi** [comment]
Sounds good. My only worry is that all rules are applied in all or nothing fashion, I could not manage to make it work with only a subset of rules. If there is a way to do it now or there are no objections to applying all rules, then let’s add it.

**Comment by sunchao** [comment]
Yes it is all or nothing, which means we need a giant commit to fix all the existing style violations. Personally I'm fine with that. I'll add a PR for the rules & results.

**Comment by sadikovi** [comment]
Okay, thanks.
